charts/external-dns/templates/deployment.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       {{- include "external-dns.selectorLabels" . | nindent 6 }}
   strategy:
     {{- toYaml .Values.deploymentStrategy | nindent 4 }}
-  {{- if or (kindIs "float64" .Values.revisionHistoryLimit) (kindIs "int64" .Values.revisionHistoryLimit) }}
+  {{- if not (has (quote .Values.revisionHistoryLimit) (list "" (quote ""))) }}
```

Comment:
It works according to my test, but I have question.
Would it make sense to write something simpler like ` if (not (empty .Values.revisionHistoryLimit))` or am I missing something ?

Reply:
Unfortunately `empty` evaluates `0` to `true` so doesn't work for numeric values where `0` is a valid value.